example/lib/main.dart

```diff
@@ -132,4 +150,8 @@ class _AtlasSampleState extends State<AtlasSample> {
       ),
     );
   }
+
+  bool _shouldShowMapTypeButton() {
+    return (AtlasProvider.instance.supportedMapTypes.length > 1);
```

Comment:
```suggestion
    return AtlasProvider.instance.supportedMapTypes.length > 1;
```

Reply:
also if `supportedMapTypes` is just needed for this logic couldn't there be a `showMapTypeButton` flag similar to the `showLocationButton` flag?

Reply:
In this example, that is all it's being used for. However, I added it for the flexibility of knowing what mapTypes the provider actually supports. 

This information could help to dynamically drive an interface like this: 
<img width="531" alt="Screen Shot 2020-01-21 at 8 57 56 AM" src="https://user-images.githubusercontent.com/944837/72785784-490c6c80-3c2c-11ea-97e7-239e87a85389.png">

It could also be used to drive a button to cycle through the available `mapTypes`, if a client wanted to have a button to loop through `.normal` > `.satellite` > `.terrain` > `.hybrid`. If you think it is helpful, I can create this experience in the example to demonstrate the capability.